tests/scripts/getbuild.py

```diff
@@ -111,17 +116,17 @@ def main():
 
     parser = argparse.ArgumentParser(description='Download artifacts from sonic azure devops.')
     parser.add_argument('--buildid', metavar='buildid', type=int, help='build id')
-    parser.add_argument('--branch', metavar='branch', type=str, help='branch name')
+    parser.add_argument('--branch', metavar='branch', type=str, default='master', help='branch name')
```

Comment:
can you also add download type option, for example the aboot versus onie image. for onie iamge, can downlaod the bin, for aboot, then download swi  image.

Reply:
also add type to download debug v.s. non debug image.